experimental/PropertyDDS/examples/schemas/src/schemasRegisterer.ts

```diff
@@ -4,8 +4,11 @@
  */
 import { ALL_SCHEMAS } from "./";
 
-export const registerSchemas = function(propertyFactory: any) {
-    Object.values(ALL_SCHEMAS).forEach((schemas) => {
+// TODO: Use something other than `any`
```

Comment:
It says that we should be specifying a type besides `any`, do we mean to work on it later on?

Reply:
Yes and no. This is definitely forward looking, and the work to do it would involve breaking changes. The primary goal with this rule (and the local overrides) is to prevent any new instances of explicit `any`, while also documenting the places where we should theoretically fix up later. I'm not explicitly tracking this work though.